app/views/admin/worldwide_offices/_worldwide_office.html.erb

```diff
@@ -46,9 +46,9 @@
   <% unless worldwide_organisation.is_main_office?(worldwide_office) %>
     <%= button_to 'Set as main office', set_main_office_admin_worldwide_organisation_path(worldwide_organisation, worldwide_organisation: {main_office_id: worldwide_office.id}), method: :put, class: 'btn btn-info' %>
     <% if worldwide_organisation.office_shown_on_home_page?(worldwide_office) %>
-      <%= button_to "Remove from home page", [:remove_from_home_page, :admin, worldwide_organisation, worldwide_office], class: 'btn btn-default' %>
+      <%= button_to "Remove from home page", [:remove_from_home_page, :admin, worldwide_organisation, worldwide_office], class: 'btn btn-default', method: :post %>
     <% else %>
-      <%= button_to "Add to home page", [:add_to_home_page, :admin, worldwide_organisation, worldwide_office], class: 'btn btn-default' %>
+      <%= button_to "Add to home page", [:add_to_home_page, :admin, worldwide_organisation, worldwide_office], class: 'btn btn-default', method: :post %>
```

Comment:
I'm a bit confused by the reasoning behind this change. How does this end up as a PATCH form? 

Looking at the source code it looks like it should only be get or post https://github.com/rails/rails/blame/0169d15bc7ec4557971d6ac6120e48b2cac9c407/actionview/lib/action_view/helpers/url_helper.rb#L342 unless I'm missing something 🤔

Reply:
Weirdly I can no longer reproduce this, so I suppose I'll just remove this change. 🤔